packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/TransactionList/index.tsx

```diff
@@ -47,11 +47,12 @@ class TransactionList extends PureComponent<Props> {
                 currency={currency}
               />
             ) : 'priceFunnel' in tx ? (
-              <SwapOrderTx order={tx} coin={coin as CoinType} />
+              <SwapOrderTx key={i} order={tx} coin={coin as CoinType} />
```

Comment:
maybe to use something else than `i`, like `tx.hash`?

Reply:
yeah that's what the first component in that list was using, but I think hash doesn't exist in the other components. I'll double check though.

Reply:
each tx has a unique `ts.id` so I used that instead.